homeassistant/components/ecobee/strings.json

```
@@ -129,6 +129,20 @@
         }
       }
     },
+    "set_aux_cutover_threshold": {
+      "name": "Set the temperature at which aux heat takes over",
+      "description": "Sets the minimum temperature at which the compressor will be used to generate heat, and at which Emergency/Aux heat will take precedence.",
+      "fields": {
+        "entity_id": {
+          "name": "Entity",
+          "description": "[%key:component::ecobee::services::set_aux_cutover_threshold::fields::entity_id::description%]"
```

This substitution is referencing itself.
In this case, since this is the only aux cutover service, the entity_id description could be a plain string describing what they're selecting. Something like; 
```suggestion
          "description": "Ecobee thermostat on which to set the aux cutover threshold."
```